package.json

```diff
@@ -106,10 +106,11 @@
     "files:flip-to-nightly:windows": "./scripts/flip-files-to-nightly.ps1",
     "files:invalidate-bucket": "./scripts/invalidate-files-bucket.sh",
     "files:invalidate-bucket:nightly": "./scripts/invalidate-files-bucket.sh --nightly",
-    "postinstall": "yarn fetch:samples && yarn xstate:typegen && ./node_modules/.bin/electron-rebuild",
+    "postinstall": "yarn xstate:typegen && ./node_modules/.bin/electron-rebuild",
```

Comment:
Unrelated to this PR but [we don't need the typegen command since we're on V5](https://stately.ai/docs/machines#typegen)

Reply:
I'll make a follow-up PR.

Reply:
https://github.com/KittyCAD/modeling-app/pull/5662